app/src/main/java/fr/gaulupeau/apps/Poche/ui/ReadArticleActivity.java

```diff
@@ -746,7 +746,13 @@ private String readRawString(int id) throws IOException {
             return sb.toString();
         } finally {
             if(reader != null) {
-                reader.close();
+                try {
+                    reader.close();
+                } catch (IOException e) {
+                    // unrecoverable exception, only log it and move on
+                    Log.w(TAG, "readRawString() failed to close reader\n" + e.toString()
+                            + "\n" + e.getStackTrace().toString());
+                }
```

Comment:
Okay, you are right. By now we do not care about this reader being closed. But we might care if we observe some strange bug. 
Does this log message have any bad side effects if we leave it in? I can also improve/reduce the message itself or change the log level, but i would not want to leave it out.